docs/modules/converter/pages/revealjs-plugins.adoc

```diff
@@ -5,10 +5,11 @@
 
 By default, generated presentations will have the following reveal.js plugins enabled:
 
-* plugin/zoom-js/zoom.js
+* plugin/zoom/zoom.js
 * plugin/notes/notes.js
+* plugin/highlight/highlight.js
```

Comment:
I don't want to enable this plugin by default. The goal of this pull request is to support reveal.js 4.x with as little changes as possible.

Before making this change, we need to discuss if we want to enable highlight.js by default. But it should be done in a follow-up issue not as part of this pull request.

Reply:
At the beginning, I used the reveal.js new highlight plugin, then I found out that reveal-asciidoctor needs its own highlight solution. So, I changed the code, and highlights.js loading behavior likes before, but not updated the doc. For now, doc has updated.